content/docs/1.8.0/snapshots-and-backups/backup-and-restore/set-backup-target.md

```diff
@@ -387,79 +402,54 @@ You can find an example CIFS backupstore for testing purpose [here](https://gith
-    # AZBLOB_ACCOUNT_KEY:    Account key of Azure Blob Storage server
-    # AZBLOB_ENDPOINT:       Endpoint of Azure Blob Storage server
-    # AZBLOB_CERT:           SSL certificate for Azure Blob Storage server
-
-    AZBLOB_ACCOUNT_NAME=`echo -n ${AZBLOB_ACCOUNT_NAME} | base64`
-    AZBLOB_ACCOUNT_KEY=`echo -n ${AZBLOB_ACCOUNT_KEY} | base64`
-    AZBLOB_ENDPOINT=`echo -n ${AZBLOB_ENDPOINT} | base64`
-    AZBLOB_CERT=`echo -n ${AZBLOB_CERT} | base64`
-
-    cat <<EOF >>azblob_secret.yml
-    apiVersion: v1
-    kind: Secret
-    metadata:
-      name: azblob-secret
-      namespace: longhorn-system
-    type: Opaque
-    data:
-      AZBLOB_ACCOUNT_NAME: ${AZBLOB_ACCOUNT_NAME}
-      AZBLOB_ACCOUNT_KEY: ${AZBLOB_ACCOUNT_KEY}
-      #AZBLOB_ENDPOINT: ${AZBLOB_ENDPOINT}
-      #AZBLOB_CERT: ${AZBLOB_CERT}
-      #HTTP_PROXY: aHR0cDovLzEwLjIxLjkxLjUxOjMxMjg=
-      #HTTPS_PROXY: aHR0cDovLzEwLjIxLjkxLjUxOjMxMjg=
-    EOF
-
-    kubectl apply -f azblob_secret.yml
-    ```
+  Set **URL**. The target URL should look like this:
 
-  - CLI command:
-    ```shell
-    kubectl create secret generic <azblob-secret> \
-      --from-literal=AZBLOB_ACCOUNT_NAME=<your-azure-storage-account-name> \
-      --from-literal=AZBLOB_ACCOUNT_KEY=<your-azure-storage-account-key> \
-      --from-literal=HTTP_PROXY=<your-proxy-ip-and-port> \
-      --from-literal=HTTPS_PROXY=<your-proxy-ip-and-port> \
-      --from-literal=NO_PROXY=<excluded-ip-list> \
-      -n longhorn-system
+    ```txt
+    azblob://[your-container-name]@core.windows.net/
     ```
 
-Then, navigate to Longhorn UI > Setting > General > Backup
-
-1. Set **Backup Target**. The target URL should look like this:
+   Make sure that you have `/` at the end, otherwise you will get an error. A subdirectory (prefix) may be used:
 
     ```txt
-    azblob://[your-container-name]@[endpoint-suffix]/
+    azblob://[your-container-name]@core.windows.net/my-path/
     ```
 
-   Make sure that you have `/` at the end, otherwise you will get an error. A subdirectory (prefix) may be used:
+  Set **Credential Secret**.
 
-    ```text
-    azblob://[your-container-name]@[endpoint-suffix]/my-path/
+    ```txt
+    longhorn-azblob-secret
     ```
 
-   - If you set `<endpoint-suffix>` in the URL, the default endpoint suffix will be `core.windows.net`.
-   - If you set `AZBLOB_ENDPOINT` in the secret, Longhorn will use `AZBLOB_ENDPOINT` as your storage URL, and `<endpoint-suffix>` will not be used even if it has been set. 
+6. 6. Create a Kubernetes secret named `longhorn-azblob-secret`.
+  This secret is used to access the backupstore in the Longhorn namespace (default: `longhorn-system`) with the following content:
 
-2. Set **Backup Target Credential Secret**
+  ```shell
+  #!/bin/bash
+  cat <<EOF >>longhorn-azblob-secret.yml
+  apiVersion: v1
+  kind: Secret
+  metadata:
+    name: longhorn-azblob-secret
+    namespace: longhorn-system
+  type: Opaque
+  stringData:
+    AZBLOB_ACCOUNT_NAME: "<Storage account name>"
```

Comment:
Don't you need to mention the following configurations anymore?

```yaml
      #AZBLOB_ENDPOINT: ${AZBLOB_ENDPOINT}
      #AZBLOB_CERT: ${AZBLOB_CERT}
      #HTTP_PROXY: aHR0cDovLzEwLjIxLjkxLjUxOjMxMjg=
      #HTTPS_PROXY: aHR0cDovLzEwLjIxLjkxLjUxOjMxMjg=
```

Reply:
Listing these parameters is a bit confusing to users
Using `AZBLOB_ACCOUNT_NAME` and `AZBLOB_ACCOUNT_KEY` parameters is enough to access the Azure blob storage server.
Added the parameters back and added the description to explain how and when to use the parameters.
Thanks